readthedocs/builds/forms.py

```diff
@@ -33,6 +40,22 @@ class Meta(object):
         model = Version
         fields = ['active', 'privacy_level', 'tags']
 
+    def clean_active(self):
+        active = self.cleaned_data['active']
+        if self._is_default_version() and not active:
+            msg = (
+                '{} is the default version of the project, '
+                'it should be active.'
+            )
+            raise forms.ValidationError(
+                _(msg.format(self.instance.verbose_name))
```

Comment:
The message should be first translated and after that, formatted.

So, the `_` function has to be added in the line 46 when the text is created.